src/Generation/UnitTestsGenerator.php

```diff
@@ -309,12 +309,49 @@ private function testExceptionalCaseLro(MethodDetails $method): PhpMethod
     {
         // TODO: Support resource-names in request args.
         // TODO: Support empty-returning RPCs
-        [$initCode] = $this->lroTestInit($method->testExceptionMethodName);
+        $status = AST::var('status');
+        $expectedExceptionMessage = AST::var('expectedExceptionMessage');
+        $requestVars = $method->requiredFields->map(fn($x) => AST::var($x->camelName));
+        $response = AST::var('response');
+        $expectedOperationsRequestObject = AST::var('expectedOperationsRequestObject');
+        $ex = AST::var('ex');
+        [$initCode, $operationsTransport, $client, $transport] = $this->lroTestInit($method->testExceptionMethodName);
         return AST::method($method->testExceptionMethodName)
             ->withAccess(Access::PUBLIC)
             ->withBody(AST::block(
                 $initCode,
-                // TODO: Complete test...
+                AST::assign($status, AST::new($this->ctx->type(Type::stdClass()))()),
```

Comment:
Would you be open to formatting like the following? At least for me it makes parsing what is going on a little easier, and seems like it may make the file a little more friendly when it needs to be edited.

```php
AST::assign(
    $status,
    AST::new($this->ctx->type(Type::stdClass()))()
),
AST::assign(
    AST::access(
        $status,
        AST::property('code')
    ),
    AST::access(
        $this->ctx->type(Type::fromName(Code::class)),
        AST::constant('DATA_LOSS')
    )
),
```

Reply:
My opinion is that I'd rather not do this, as it will cause the source code to get considerably longer.
I suggest leaving as it is for now, but we can always reformat later if it becomes a problem.